### ISSUE TITLE
Issue running command [gcloud beta dataproc jobs list] with [--state-filter active] arguments

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prototipando/google-cloud-sdk#397]
```
Issue running command [gcloud beta dataproc jobs list].

What steps will reproduce the problem?
Provide the flags [--state-filter active]

What is the expected output? What do you see instead?
I expect to see a list of all active jobs, instead I get the following error
```

> gcloud beta dataproc jobs list --state-filter active  
> ERROR: gcloud crashed (AttributeError): type object 
> 'DataprocProjectsJobsListRequest' has no attribute 'JobStateMatcher'

If you would like to report this issue, please run the following command:
  gcloud feedback

```

Please provide any additional information below.


Traceback (most recent call last):
  File "calliope/cli.py", line 609, in Execute
    result = args.cmd_func(cli=self, args=args)
  File "calliope/backend.py", line 1228, in Run
    resources = command_instance.Run(args)
  File "api_lib/dataproc/util.py", line 61, in CatchHTTPErrorRaiseHTTPException
    return func(*args, **kwargs)
  File "surface/dataproc/jobs/list.py", line 75, in Run
    messages.DataprocProjectsJobsListRequest.JobStateMatcher.ACTIVE)
AttributeError: type object 'DataprocProjectsJobsListRequest' has no attribute 
'JobStateMatcher'

Installation information:

Google Cloud SDK [0.9.87]

Platform: [Mac OS X, x86_64]
Python Version: [2.7.10 (default, Aug 22 2015, 20:33:39)  [GCC 4.2.1 Compatible 
Apple LLVM 7.0.0 (clang-700.0.59.1)]]
Python Location: [/usr/bin/python]
Site Packages: [Disabled]

Installation Root: [/Users/gilj/google-cloud-sdk]
Installed Components:
  core: [2015.11.18]
  core-nix: [2015.11.16]
  gcloud: []
  gsutil-nix: [4.15]
  beta: [2015.10.08]
  gsutil: [4.15]
  bq: [2.0.18]
  alpha: [2015.10.08]
  bq-nix: [2.0.18]
System PATH: 
[/Users/gilj/google-cloud-sdk/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin]
Cloud SDK on PATH: [True]

Installation Properties: [/Users/gilj/google-cloud-sdk/properties]
User Config Directory: [/Users/gilj/.config/gcloud]
User Properties: [/Users/gilj/.config/gcloud/properties]
[output truncated]
```

Original issue reported on code.google.com by `gilles...@gmail.com` on 20 Nov 2015 at 3:36

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting this. We've successfully reproduced the issue and are 
working on a fix. We'll keep this issue updated with any developments.
```

Original comment by `z...@google.com` on 24 Nov 2015 at 5:10
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This should have been fixed in the Cloud SDK 90.0.0 release. Sorry for the slow 
update.
```

Original comment by `z...@google.com` on 4 Jan 2016 at 2:11
- Changed state: **Fixed**
